python/pp_ser.py

```diff
@@ -564,46 +563,26 @@ def __ser_off(self, args):
     def __re_module(self):
         r = re.compile('^ *(module|program) +([a-z][a-z0-9_]*)', re.IGNORECASE)
         m = r.search(self.__line)
+
         if m:
             if m.group(2).upper() == 'PROCEDURE':
                 return False
             if self.__module:
                 self.__exit_error(msg = 'Unexpected ' + m.group(1) + ' statement')
+            if(m.group(1).upper() == 'MODULE'):
+                self.__useStmtInModule = True
```

Comment:
Also what is the scope of this function? I also see a `self.__module` case above.